python/IECoreRI/TestCase.py

```diff
@@ -47,4 +47,4 @@ def setUp( self ) :
 	def tearDown( self ) :
 		
 		if os.path.isdir( "test/IECoreRI/output" ) :
-			shutil.rmtree( "test/IECoreRI/output" )
+			shutil.rmtree( "test/IECoreRI/output", True )
```

Comment:
`ignore_errors = True` would be more self explanatory

Reply:
Addressed in ae0288f